test/Microsoft.DotNet.PackageValidation.Tests/Filtering/TargetFrameworkFilterTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.PackageValidation.Filtering;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.PackageValidation.Tests.Filtering
+{
+    public class TargetFrameworkFilterTests
+    {
```

Comment:
Can you also please add a test for the exact case that prompted the creation of this PR? `portable-net45+win8+wpa81+wp8`

Relevant thread: https://github.com/dotnet/maintenance-packages/pull/38/files#r1471429706

Reply:
An additional test in the BaselinePackageValidatorTests.cs file would make sense too for this case, since that's the specific scenario that prompted this PR.